### PR TITLE
feat: global bridge machines

### PR DESCRIPTION
**Pull request by LuizAsFight**
Closes
FRO-496
FRO-497

This PR addresses a refactor to add usage of global machines but for dynamic "spawned actors". this allows us to keep same reference to a bridging(deposit/withdraw) process happenning, instead of needing to instantiate a new machine every time we need

### CODE REVIEW

File: packages/app/src/systems/Bridge/components/BridgeTxItem/BridgeTxItem.tsx [hunk 1]
```diff
@@ -7,10 +7,10 @@ type BridgeTxItemProps = {
...
   fromLogo: ReactNode;
   toLogo: ReactNode;
-  asset: {
-    assetImageSrc: ReactNode | string;
-    assetAmount: string;
-    assetSymbol: string;
+  asset?: {
+    assetImageSrc?: ReactNode | string;
+    assetAmount?: string;
+    assetSymbol?: string;
```
Comment (luizstacio) [inline packages/app/src/systems/Bridge/components/BridgeTxItem/BridgeTxItem.tsx:13]: Why we asset become optional right now?

File: packages/app/src/systems/Bridge/services/bridge.tsx [hunk 2]
```diff
@@ -69,9 +81,17 @@ export class BridgeService {
       });
 
       if (txId) {
-        store.openTxEthToFuel({
-          txId,
-        });
+        if (fuelWallet) {
```
Comment (luizstacio) [inline packages/app/src/systems/Bridge/services/bridge.tsx:84]:
```suggestion
        if (txId && fuelWallet) {
```
You don't need to have to nested ifs in this case

**Review by luizstacio — CHANGES_REQUESTED** [review]

File: packages/app/src/systems/Bridge/components/BridgeTxItem/BridgeTxItem.tsx (hunk 1 above)
Comment (LuizAsFight) [reply packages/app/src/systems/Bridge/components/BridgeTxItem/BridgeTxItem.tsx:13]: it's just to improve flexibility and avoid needing conditionals in other place, as the asset can be undefined and will be set only after getting the tx info

File: packages/app/src/systems/Bridge/services/bridge.tsx (hunk 2 above)
Comment (LuizAsFight) [reply packages/app/src/systems/Bridge/services/bridge.tsx:84]: it's not the same logic, look the `return;  ` in the lines below

Comment (luizstacio) [reply packages/app/src/systems/Bridge/services/bridge.tsx:84]: In this case the return is outside of the `f (txId)`

**Review by luizstacio — APPROVED** [review]

**Pull request merged**